Gems/ScriptCanvas/Code/Editor/View/Widgets/NodePalette/NodePaletteModel.cpp

```diff
@@ -573,32 +592,17 @@ namespace
             const AZ::BehaviorEBusHandler::EventArray& events(handler->GetEvents());
             if (!events.empty())
             {
-                AZStd::string translationContext = ScriptCanvasEditor::TranslationHelper::GetContextName(ScriptCanvasEditor::TranslationContextGroup::EbusHandler, behaviorEbus.m_name);
-                AZStd::string categoryPath;
 
-                {
-                    AZStd::string translationKey = ScriptCanvasEditor::TranslationHelper::GetClassKey(ScriptCanvasEditor::TranslationContextGroup::EbusHandler, behaviorEbus.m_name, ScriptCanvasEditor::TranslationKeyId::Category);
-                    AZStd::string translatedCategory = QCoreApplication::translate(translationContext.c_str(), translationKey.c_str()).toUtf8().data();
+                GraphCanvas::TranslationKey key;
+                key << "EBusHandler" << behaviorEbus.m_name.c_str() << "details";
```

Comment:
Do you want a space surrounding the name? "EBusHandler " and " details"..

Reply:
no, there is no whitespace, the << operator does the concatenation in the format required by translation keys. The above resolves to "EBusHandler.EbusName.details"